tests/test_updater.py

```diff
@@ -549,6 +552,16 @@ async def set_webhook(*args, **kwargs):
             response = await send_webhook_message(ip, port, None, "TOKEN", get_method="HEAD")
             assert response.status_code == HTTPStatus.METHOD_NOT_ALLOWED
 
+            # Returns Forbidden if no secret token is set
+            response = await send_webhook_message(ip, port, update.to_json(), "TOKEN")
+            assert response.status_code == HTTPStatus.FORBIDDEN
```

Comment:
in case harshils suggestion for the "reason" is added, it would be nice to check this here